.github/workflows/cpu-inference.yml

```diff
@@ -2,13 +2,15 @@ name: cpu-inference
 
 on:
   pull_request:
-    paths-ignore:
-      - 'docs/**'
-      - 'blogs/**'
+    paths:
+      - '.github/workflows/cpu-inference.yml'
+      - 'requirements/**'
+      - 'deepspeed/inference/**'
   workflow_dispatch:
   merge_group:
     branches: [ master ]
-
+  schedule:
+        - cron: "0 0 * * 0"
```

Comment:
This runs weekly - should we have it do nightly instead?

Reply:
Discussed offline, leaving it at weekly for now, can move to nightly if needed.